cohortextractor/query_utils.py

```diff
@@ -16,11 +16,11 @@ def get_cohort_variables(cohort):
     ]
 
 
-def get_column_definitions(cohort_class):
-    if isinstance(cohort_class, CohortConcept):
-        variables = get_cohort_variables(cohort_class)
+def get_column_definitions(cohort):
+    if isinstance(cohort, CohortConcept):
```

Comment:
`CohortConcept` will be a particularly bad name if/when we rename the `concepts` module

Reply:
ho ho